src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.props.in

```diff
@@ -16,6 +16,9 @@
 		<AndroidSdkPlatformToolsVersion Condition="'$(AndroidSdkPlatformToolsVersion)' == ''">27.0.1</AndroidSdkPlatformToolsVersion>
 		<AndroidSdkToolsVersion Condition="'$(AndroidSdkToolsVersion)' == ''">26.1.1</AndroidSdkToolsVersion>
 		<AndroidNdkVersion Condition="'$(AndroidNdkVersion)' == ''">16.1</AndroidNdkVersion>
+
+		<!-- Enable nuget package conflict resolution -->
+		<ResolveAssemblyConflicts>true</ResolveAssemblyConflicts>
```

Comment:
Can the commit message include an example of what `$(ResolveAssemblyConflicts)` actually does? Can it mention how it works? e.g. it needs to be set at _evaluation_ time before `Microsoft.CSharp.targets` are imported.